model/project.go

```diff
@@ -170,6 +183,17 @@ func (p *InternalProject) Save() ([]datastore.Property, error) {
 		*version = p.Version.String()
 	}
 
+	// blueMonday policy building: https://github.com/microcosm-cc/bluemonday#policy-building
+	bmUSC := bluemonday.UGCPolicy()
+	bmUSC.AllowImages()
+	bmUSC.AllowAttrs("src").OnElements("img")
+
+	bmStrict := bluemonday.StrictPolicy()
+
+	sanitizedTitle := bmStrict.Sanitize(p.Title)
```

Comment:
If it is not too much trouble I would really like to see a test where the description/title/readme is sanitized.